### PR TITLE
ISDK-935 Update pod spec

### PR DESCRIPTION
**Pull request by spookyd**
- Adds dependencies to pod spec

### CODE REVIEW

**Review by jlebajan — APPROVED** [review]
LGTM.

File: Package.swift [hunk 1]
```diff
@@ -15,7 +15,7 @@ let package = Package(
     ],
     targets: [
         .binaryTarget(name: "FireworkVideoGAMSupport",
-                      url: "https://github.com/loopsocial/firework_ios_sdk_gam_support/releases/download/v0.1.0/FireworkVideoGAMSupport-v0.1.0.xcframework.zip",
+                      url: "https://github.com/loopsocial/firework_ios_sdk_gam_support/releases/download/v0.2.0/FireworkVideoGAMSupport-v0.2.0.xcframework.zip",
                       checksum: "7a3816a368926af23ab279522ab3cfd341d9215211be22d1a296f3eb93f20d68"),
```
Comment (jlebajan) [inline Package.swift:19]: Missed to update the checksum?

Comment (spookyd) [reply Package.swift:19]: It is the same binary. No change to checksum.

Comment (jlebajan) [reply Package.swift:19]: ✅

**Pull request merged**